### PR TITLE
my second commit

### PR DESCRIPTION
**Pull request by RnHuB**
please accept it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,4 +19,6 @@ Instructions to contributors:
 
 @rnhub
 
+@rnhub
```
Comment (Codanv) [inline README.md:22]: duplicate username

**Review by Codanv — APPROVED** [review]
You cannot add duplicate username.

File: README.md (hunk 1 above)
Comment (RnHuB) [reply README.md:22]: but this is not in readme instructions

Comment (Codanv) [reply README.md:22]: okay! you may pull request for this

Comment (RnHuB) [reply README.md:22]: thank u sir

**Comment by Codanv** [comment]
duplicate username

**Pull request closed without merging**